client/templates/requests/requests.html

```diff
@@ -63,6 +63,9 @@
 							{{#if searchType 'TV Shows'}}
                            <h4> <li><a href="{{link}}" target="_blank"><strong>{{title}} ({{year}})</strong></a></li></h4>
 							{{/if}}
+						{{#if searchType 'TV Shows'}}
```

Comment:
There's already a TV show conditional above and below this, combine with the one above or all 3?